README.md

```diff
@@ -69,7 +79,7 @@ $ make deploy
 ## 5. SSH to your instance
 
 ```bash
-$ ssh remote-workstation-<value of IDENTIFIER>
+$ make ssh-to-instance
```

Comment:
Is this so the user doesn't have to remember where the instance is, but rather it's written in a local config? or is it still in the system-wide ssh config?

Reply:
We write to either the local config or where they specify `SSH_CONFIG_LOCATION`

`make ssh-to-instance` now means that you don't need to provide anything else.

If `SSH_CONFIG_LOCATION` is populated, it'l use that and `IDENTIFIER` if that's populated, else we default to the locally created one and `dev`.

You shouldn't need nor care where the instance is